htdocs/main.inc.php

```diff
@@ -421,6 +421,19 @@ function analyseVarsForSqlAndScriptsInjection(&$var, $type)
 	}
 }
 
+// Set current modulepart
+$modulepart = explode("/", $_SERVER["PHP_SELF"]);
+if(is_array($modulepart) && count($modulepart)>0)
+{
+	foreach($conf->modules as $module)
+	{
+		if(in_array($module, $modulepart))
+		{
+			$conf->modulepart = $module;
```

Comment:
Why not adding a break; just after that ?

Reply:
sorry for this delay!
Yes you are right!